editor/int/syntax/int.vim

```diff
@@ -8,27 +8,44 @@ syntax region intCommentLine start=";" end="$" display contains=intTodo
 
 syntax match intSpecial contained "\\[nr\\\"\'t]" display
 
-syntax region intString start="\"" skip="\\\"" end="\"" contains=intSpecial
+syntax match intFormat contained "%[csSCdiuDIUZzxXpbTF%m]" display
+" TODO: Not sure about these
+" syntax match intFormat contained "%\d" display
+" syntax match intFormat contained "%\d\d" display
+" syntax match intFormat contained "%\d\d\d" display
+
+syntax region intString start="\"" skip="\\\"" end="\"" contains=intSpecial,intFormat
```

Comment:
This should be left as is seeing as these format specifiers are not part of Intercept (see also my comment on the main pr conversation).